component/registration_form.jsx

```diff
@@ -0,0 +1,44 @@
+import { TYPE_JSON } from '../api/defs'
```

Comment:
always put `.js` let's not rely on node behavior and make compatible code

Reply:
Ok